CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## 1.14.0 - 2024-03-20
+
+### Added
+* Support for creating containerized function apps [[3943]](https://github.com/microsoft/vscode-azurefunctions/pull/3943), [[3964]](https://github.com/microsoft/vscode-azurefunctions/pull/3964)
```

Comment:
nit: Could you put the issue links as prefixes like the 1.1.3 release?

Reply:
Yah I just did it this way since we have been doing it like this for other extensions but I can change it.

Reply:
Alex and I are trying to make more PM friendly notes 😋